### PR TITLE
fix: cassandra.cluster can not support multiple IPs

### PR DESCRIPTION
**Pull request by xiaojingchen**
this PR fixes: #106

### CODE REVIEW

**Comment by xiaojingchen** [comment]
@Little-Wallace PTAL

**Comment by siddontang** [comment]
em, I don't like this way, it is too verbose.
If we have multi args, we need to pass like `-p tikv.pd` many times. maybe we can use another separator, like ';' instead.

**Comment by xiaojingchen** [comment]
If `;` is not included in `""`, the value will still be parsing failure.  bash will parse `;` as the end of the command.
```
./bin/go-ycsb load cassandra -p cassandra.cluster=127.0.0.1:9042;127.0.0.2:9042
create db cassandra failed gocql: unable to create session: unable to discover protocol version: dial tcp 127.0.0.1:9042: connect: connection refused
zsh: command not found: 127.0.0.2:9042
```
the other separator like `/` `:`  may have ambiguities.
Although this way is repetitive and redundant, it is more reliable.

File: cmd/go-ycsb/main.go [hunk 1]
```diff
@@ -85,9 +85,18 @@ func initialGlobal(dbName string, onProperties func()) {
 		globalProps = properties.MustLoadFiles(propertyFiles, properties.UTF8, false)
 	}
 
+	propsMap:=map[string][]string{}
```
Comment (siddontang) [inline cmd/go-ycsb/main.go]: please format your code

File: cmd/go-ycsb/main.go [hunk 2]
```diff
@@ -85,9 +85,18 @@ func initialGlobal(dbName string, onProperties func()) {
...
-		globalProps.Set(seps[0], seps[1])
+		if propVals,exist:=propsMap[seps[0]];exist{
+			propsMap[seps[0]]=append(propVals,seps[1])
+		}else{
+			propsMap[seps[0]]=[]string{seps[1]}
+		}
+	}
+
+	for key,val:=range propsMap{
+		globalProps.Set(key, strings.Join(val,","))
```
Comment (siddontang) [inline cmd/go-ycsb/main.go]: seem there is a problem if I pass `-p operationcount=100 -p operationcount=1000.

Comment (xiaojingchen) [reply cmd/go-ycsb/main.go]: It's really a problem. just I can't find a good solution.
And I don't know why to use "-p", if not, can we avoid these problems?

**Pull request closed without merging**